promptsource/templates/lama/google_re/templates.yaml

```diff
@@ -0,0 +1,67 @@
+dataset: lama
+subset: google_re
+templates:
+  89ba7edc-d980-4431-8f82-9c07d28d1089: !Template
+    answer_choices: null
+    id: 89ba7edc-d980-4431-8f82-9c07d28d1089
+    jinja: 'Given the following sentence: {{masked_sentence | replace(''[MASK]'',
+      obj_label) }} Rewrite the following template with replacing X and Y with the
+      correct words. {{template}} |||  {{template | replace(''[X]'', sub_label) |
+      replace(''[Y]'', obj_label)}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - BLEU
+      original_task: false
+    name: X_and_Y
+    reference: ''
+  9f787820-5443-4778-ba23-5fd9f894233c: !Template
+    answer_choices: null
+    id: 9f787820-5443-4778-ba23-5fd9f894233c
+    jinja: '{{masked_sentence}} Fill the mask with {{ pred | replace("/people/person/","")  |
```

Comment:
Based on prompt source viewer it says "Fill the mask with date of birth."  The answers are year of birth?